packages/composite-checkout/src/lib/payment-methods/existing-credit-card.js

```diff
@@ -345,18 +254,25 @@ function ExistingCardPayButton( { disabled, id } ) {
 	);
 }
 
-function ExistingCardSummary( { cardholderName, brand, last4 } ) {
+function ExistingCardSummary( { cardholderName, cardExpiry, brand, last4 } ) {
+	const localize = useLocalize();
+
 	return (
 		<SummaryDetails>
 			<SummaryLine>{ cardholderName }</SummaryLine>
 			<SummaryLine>
-				{ brand !== 'unknown' && `****${ last4 }` }{ ' ' }
 				<CardFieldIcon brand={ brand } isSummary={ true } />
+				<CardDetails>**** { last4 }</CardDetails>
+				{ `${ localize( 'Exp:' ) } ${ formatDate( cardExpiry ) }` }
```

Comment:
"Exp:" is probably hard to translate. We should add some context. I don't currently have the `lib/localize.js` system set up for that but we can do it pretty easily. Just change `useLocalizeFactory` so that it looks like this:

```js
function useLocalizeFactory( locale ) {
	const localize = useRef( text => text );
	useEffect( () => {
		setLocaleData( getLocaleDataForLocale( locale ) );
		localize.current = (text, context) => _x( text, context || '', 'default' );
	}, [ locale ] );
	return localize.current;
}
```

Then you can change this to:

```suggestion
				{ `${ localize( 'Exp:', 'Credit card expiry date abbreviation' ) } ${ formatDate( cardExpiry ) }` }
```

Reply:
I'll just change the copy to say "Expiry:" It's not that long that it'll make a difference and it'll be easy to translate.